### PR TITLE
add test init command

### PR DESCRIPTION
**Pull request by roneli**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -46,9 +46,12 @@ jobs:
       - name: Build
         run: go build -v .
 
-      - name: Sanity
+      - name: Sanity Fetch
         run: go run ./main.go fetch --config=internal/test/test_config.hcl --enable-console-log
 
+      - name: Sanity Init
+        run: go run ./main.go init test --config=test_init_config.hcl
```
Comment (michelvocks) [inline .github/workflows/test.yml:53]: What does `--config` do here for init?

**Review by michelvocks — APPROVED** [review]
LGTM. Just a small question.

File: .github/workflows/test.yml (hunk 1 above)
Comment (roneli) [reply .github/workflows/test.yml:53]: instead of creating config.hcl I give it a different name, this is to control where the config is created

Comment (michelvocks) [reply .github/workflows/test.yml:53]: Gotcha. Shouldn't fetch then also use the same config? The name is currently different.

**Pull request merged**